docs/using_lagoon/index.md

```diff
@@ -48,7 +48,7 @@ Some Docker images and containers need additional customizations from the provid
 
 | Type | Versions | Dockerfile |
 | :--- | :--- | :--- |
-| [Elasticsearch](docker_images/elasticsearch.md) | 6.8, 7.1, 7.3 | [elasticsearch/Dockerfiles](https://github.com/amazeeio/lagoon/tree/master/images/elasticsearch) |
+| [Elasticsearch](docker_images/elasticsearch.md) | 6.8, 7.1, 7.6 | [elasticsearch/Dockerfiles](https://github.com/amazeeio/lagoon/tree/master/images/elasticsearch) |
```

Comment:
let's remove the `7.1` ELK images with this PR as we don't need them anymore

Reply:
Remove by latest commit